Customizing/global/plugins/Services/Repository/RepositoryObject/ReportTrainerOpTepCat/classes/class.ilObjReportTrainerOpTepCat.php

```diff
@@ -42,108 +43,237 @@ public function getRelevantParameters() {
 	}
 
 	protected function buildQuery($query) {
-		$query 	->select("hu.user_id")
-				->select_raw("CONCAT(hu.lastname, ', ', hu.firstname) as fullname");
+		return null;
+	}
+
+	protected function fetchData(callable $callback) {
```

Comment:
Ich fände es schöner, wenn du die query innerhalb der methode buildQueryStatement implementiers, und hier nur die fertige query qua dieser methode holst, dann kann man nämlich die gleich query auch in der query view laden